inc/cleantalk-public-integrations.php

```diff
@@ -464,18 +464,31 @@ function ct_woocommerce_checkout_check()
     $post_info['comment_type'] = 'order';
     $post_info['post_url']     = apbct_get_server_variable('HTTP_REFERER');
 
-    //Making a call
-    $base_call_result = apbct_base_call(
-        array(
-            'message'         => $message,
-            'sender_email'    => $sender_email,
-            'sender_nickname' => $sender_nickname,
-            'post_info'       => $post_info,
-            'js_on'           => apbct_js_test('ct_checkjs', $_COOKIE, true),
-            'sender_info'     => array('sender_url' => null),
-        )
+    $base_call_data = array(
+        'message'         => $message,
+        'sender_email'    => $sender_email,
+        'sender_nickname' => $sender_nickname,
+        'post_info'       => $post_info,
+        'js_on'           => apbct_js_test('ct_checkjs', $_COOKIE, true),
+        'sender_info'     => array('sender_url' => null)
     );
 
+    /**
+     * Add honeypot_field to $base_call_data is forms__wc_honeypot on
+     */
+    if ( isset($apbct->settings['forms__wc_honeypot']) && $apbct->settings['forms__wc_honeypot'] ) {
```

Comment:
isset() is redundant. You could use 
`$apbct->settings['forms__wc_honeypot']`

Reply:
Fixed